src/XProfile/VirtoCommerce.XProfile/Schemas/InputUpdateContactType.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.Types;
+using VirtoCommerce.CustomerModule.Core.Model;
+
+namespace VirtoCommerce.ExperienceApiModule.XProfile.Schemas
+{
+    public class InputUpdateContactType : InputMemberBaseType
+    {
+        public InputUpdateContactType()
+        {
+            Field<NonNullGraphType<StringGraphType>>(nameof(Contact.Id));
```

Comment:
the code duplication may be reduced by defining the base InputType

Reply:
ok